awsconfigs/common/aws-alb-ingress-controller/base/deployment_patch.yaml

```diff
@@ -1,18 +1,16 @@
-# Application Load Balancer (ALB) Ingress Controller Deployment Manifest.
-# This manifest details sensible defaults for deploying an ALB Ingress Controller.
-# GitHub: https://github.com/kubernetes-sigs/aws-alb-ingress-controller
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: alb-ingress-controller
+  name: aws-load-balancer-controller
```

Comment:
why this name change?

Reply:
matching upstream
> This project was formerly known as "AWS ALB Ingress Controller", we rebranded it to be "AWS Load Balancer Controller".

https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/